net-misc/gns3-gui/gns3-gui-2.2.36.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..10} )
+PYTHON_COMPAT=( python3_{10..11} )
```

Comment:
Did upstream remove python 3.9 support?  For now only python 3.8 is masked and about to be removed, python 3.9 is still in tree and supported